ibm_catalog.json

```diff
@@ -400,27 +401,25 @@
             }
           ],
           "architecture": {
-            "descriptions": "This architecture supports the deployment of Log Analysis and Cloud Monitoring instances on IBM Cloud. It also supports configuring the Activity Tracker event routing.",
+
+            "descriptions": "This architecture supports the deployment of IBM Log Analysis and IBM Cloud Monitoring instances on IBM Cloud. It also supports the deployment of IBM Cloud Activity Tracker Event Routing to either an Object Storage bucket or Log Analysis target.",
```

Comment:
I think this would be better since both the object storage bucket and log analysis can be set as AT targets together and `either` `or` was giving the impression that only one at a time can be routed.

`This architecture supports the deployment of IBM Log Analysis and IBM Cloud Monitoring instances on IBM Cloud. It also supports the deployment of IBM Cloud Activity Tracker Event Routing to an Object Storage bucket and Log Analysis target.`